operator/gce.go

```diff
@@ -42,6 +42,9 @@ type GCEShutdownCall struct {
 // create instance list
 func valuesGCE(m map[string]compute.InstancesScopedList) []*compute.Instance {
 	var res []*compute.Instance
+	if m == nil {
```

Comment:
🔧I thing this needsto be changed.
This code should be unnecessary because this is illegal that map is nil.
Please investigate the cause of map becoming nil.